lib/scm-data-generators/git.js

```diff
@@ -0,0 +1,27 @@
+var CoreObject  = require('core-object');
+var Promise     = require('ember-cli/lib/ext/promise');
+var gitRepoInfo = require('git-repo-info');
+var simpleGit = require('simple-git');
+
+module.exports = CoreObject.extend({
+  init: function(path) {
+    this.path = path;
+  },
+
+  generate: function() {
+    var _this = this;
+    return new Promise(function(resolve, reject) {
+      simpleGit(_this.path).log(function(err, log) {
+        var info = log.latest;
+        var sha = info.hash.slice(0, 7);
```

Comment:
why not preserve the whole hash? Other plugins that want to shorten it can choose to do that